### PR TITLE
Bugfixes + Ship Namelist Expansion

### PR DESCRIPTION
**Pull request by MarkSuckerberg**
## About The Pull Request
Adds a **boat**load of new ship names to the random generator list, as well as fixes both the bug where damaged ships were invisible when highly damaged and lowers the minimum threshold for ship naming/large icon use.

## Why It's Good For The Game
Bugfixes good, new randomgen names also good

## Changelog
:cl:
add: Many more randomgen ship names.
fix: Critically damaged ships are no longer invisible.
tweak: Small ships such as the kilo now qualify for names and full ship icons. Sorry pill-class "users," you don't get nice things.
/:cl:

### CODE REVIEW

File: strings/ship_names.txt [hunk 1]
```diff
@@ -1,20 +1,166 @@
...
+Eagle
+Edward
+Eisenhower
+Emerald
+Erudite
+Essex
+Evergreen
+Exception Handler
+Exile
+fairy
```
Comment (triplezeta) [inline strings/ship_names.txt]: fairy

File: strings/ship_names.txt [hunk 2]
```diff
@@ -1,20 +1,166 @@
...
+Falchion
 Falcon
+feather
```
Comment (triplezeta) [inline strings/ship_names.txt]: feather

File: strings/ship_names.txt (hunk 1 above)
Comment (ZephyrTFA) [inline strings/ship_names.txt]:
```suggestion
Fairy
```

File: strings/ship_names.txt (hunk 2 above)
Comment (ZephyrTFA) [inline strings/ship_names.txt]:
```suggestion
Feather
```

File: strings/ship_names.txt [hunk 3]
```diff
@@ -1,20 +1,166 @@
...
+Goldfish
+Grim
+Gryphon
+Halberd
+Hammerhead
+Hicular Manslaughter
+Honest Mistake
 Hound
-Hyperborean
+How's My Driving?
```
Comment (ZephyrTFA) [inline strings/ship_names.txt:100]: terrible, this is why kobolds arent given ship licenses

**Review by ZephyrTFA — CHANGES_REQUESTED** [review]

**Review by ZephyrTFA — APPROVED** [review]

**Pull request merged**